arch/arm/core/aarch32/cortex_m/fault.c

```diff
@@ -675,7 +676,8 @@ static uint32_t hard_fault(z_arch_esf_t *esf, bool *recoverable)
 
 	if ((SCB->HFSR & SCB_HFSR_VECTTBL_Msk) != 0) {
 		PR_EXC("  Bus fault on vector table read");
-	} else if ((SCB->HFSR & SCB_HFSR_FORCED_Msk) != 0) {
+	}
+	if ((SCB->HFSR & SCB_HFSR_FORCED_Msk) != 0) {
```

Comment:
This does change the behavior, slightly, right? Cause originally, if the fault is on Vector Table read, the HF escalation conditional is not processed.

Reply:
Thank you! I misunderstood this slightly. I will revert the change to preserve the logic. Looks like I need to go with a final non-empty else{} (see other comment for our resolution on contents).

Reply:
Thanks for the fix.